packages/analytics-core/src/timeline.ts

```diff
@@ -31,13 +31,20 @@ export class Timeline {
     return Promise.resolve();
   }
 
+  reset() {
+    this.plugins = [];
```

Comment:
should we also reset `applying` and `flushing`? i think if there's no plugins those ops will be no-op

Reply:
Good point! I don't see any usage of `flushing`, so I will remove this flag variable. For `applying`, I will reset it too.